perf/src/sigverify.rs

```diff
@@ -38,6 +38,7 @@ const TRACER_KEY_BYTES: [u8; 32] = [
     155, 90, 30, 39, 116, 115, 238, 38, 126, 21, 232, 133,
 ];
 const TRACER_KEY: Pubkey = Pubkey::new_from_array(TRACER_KEY_BYTES);
+const TRACER_KEY_OFFSET_IN_TRANSACTION: usize = 69;
```

Comment:
@pieceofr this will eventually be integrated with the ping service, so I'm going to verify this works with those pings on a local testnet. It also means our pings:

1. Have to use this new `TRACER_KEY` as a fee payer
2. Will need to be a fixed type of transaction where the fee payer is present in the transaction at offset `TRACER_KEY_OFFSET_IN_TRANSACTION`

Reply:
@pieceofr I verified using your mini-ping server that the fee payer here is also at offset 69, so this should be good to go!